cashweb-payload/proto/payload.proto

```diff
@@ -47,3 +47,9 @@ message BurnTx {
     // the signed payload.
     uint32 burn_idx = 2;
 }
+
+// Set of auth wrappers for returning multiple items to the client as needed.
+message SignedPayloadSet {
+    // Set of auth wrappers that can be used in certain get responses
+    repeated SignedPayload items = 1;
+}
```

Comment:
Need a pointer to fetch the next page.

Reply:
Need to fix this indentation here. VS Code doesn't respect indentation settings.